proto/compat/repair_utf8_gen.go

```diff
@@ -0,0 +1,2754 @@
+// Code generated by cmd/tools/genvisitor. DO NOT EDIT.
+
+package compat
+
+import (
+	apicommand "github.com/temporalio/s2s-proxy/proto/1_22/api/command/v1"
+	apifailure "github.com/temporalio/s2s-proxy/proto/1_22/api/failure/v1"
+	apihistory "github.com/temporalio/s2s-proxy/proto/1_22/api/history/v1"
+	apiupdate "github.com/temporalio/s2s-proxy/proto/1_22/api/update/v1"
+	apiworkflow "github.com/temporalio/s2s-proxy/proto/1_22/api/workflow/v1"
+	apiworkflowservice "github.com/temporalio/s2s-proxy/proto/1_22/api/workflowservice/v1"
+	serverapiadminservice "github.com/temporalio/s2s-proxy/proto/1_22/server/api/adminservice/v1"
+	serverapihistory "github.com/temporalio/s2s-proxy/proto/1_22/server/api/history/v1"
+	serverapipersistence "github.com/temporalio/s2s-proxy/proto/1_22/server/api/persistence/v1"
+	serverapireplication "github.com/temporalio/s2s-proxy/proto/1_22/server/api/replication/v1"
+)
+
+func repairInvalidUTF8(vAny any) (ret bool, retErr error) {
```

Comment:
is repairInvalidUTF8 hooked in the grpc codec already or will be a separate PR?

Reply:
Right, just splitting up PRs a little.